localisation/00_zuwc_united_localisation.csv

```diff
@@ -2920,16 +2920,17 @@ EVTOPTA_WCPRM_600;As water wish.;;Wie Wasser w
 EVTOPTA_WCPRM_605;Old fool!;;Alter Narr!;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+form_the_gorian_empire;Form the Gorian Empire;;Gr�ndet das Gorianische Imperium;;;;;;;;;;;x
+form_the_gorian_empire_desc;Once, the Gorian Empire was the greatest state on Draenor. It is time to bring its legacy and glory to the brave new world.;;Einst war das gorianische Imperium die gr��te Nation auf Draenor. Es ist an der Zeit, sein Verm�chtnis und seinen Ruhm in die sch�ne neue Welt zu bringen.;;;;;;;;;;;x
+has_realm_province_with_5_or_more_settlements_tooltip;Has 5 or more holdings in your realm. ;;Verf�gt �ber 5 oder mehr Besitzungen in Eurem Reich. ;;;;;;;;;;;x
+create_gorian_empire_tt;�Y[Root.GetBestName]�! creates the �Y[e_gorian_empire.GetName]�!;;�Y[Root.GetBestName]�! gr�ndet das �Y[e_gorian_empire.GetName]�!;;;;;;;;;;;x
+EVTTITLE_WCOE_100;Reborn in Purple;;In Purpur wiederauferstanden;;;;;;;;;;;x
+EVTDESC_WCOE_100;"If we can't return to Draenor, we will rebuild our civilisation in full glory in our new world!", with these words [target_imperator.GetTitledFirstName] started [target_imperator.GetHerHis] speech to the young senate and people of [target_new_goria.GetName].\n\nPurple flags were hung all over the capital earlier that day in preparation of this unique event. The central square was surrounded by purple and gold banners to emphasize the glory of the past and future empire. The Senate declared the beginning of a new era upon Azeroth, the Gorian Era.\n\nThe masses assembled in the square in front of the Senate while they exclaimed loudly, saluting their first new [target_imperator.GetTitle] since their departure from Draenor.;;�Wenn wir nicht nach Draenor zur�ckkehren k�nnen, werden wir unsere Zivilisation in unserer neuen Welt in vollem Glanz wieder aufbauen.� Mit diesen Worten begann [target_imperator.GetTitledFirstName] [target_imperator.GetHerHis]e Rede vor dem jungen Senat und den Einwohnern von [target_new_goria.GetName].\n\nPurpurne Flaggen wurden fr�her am Tag in der ganzen Hauptstadt zur Vorbereitung dieses einzigartigen Ereignisses gehisst. Der Hauptplatz war von purpurroten und goldenen Bannern umgeben, um den Ruhm des vergangenen und zuk�nftigen Reiches zu demonstrieren.Der Senat erkl�rte den Beginn einer neuen �ra in Azeroth, der gorianischen �ra.\n\nDie Massen versammelten sich auf dem Platz vor dem Senat, lauthals johlend und ihren ersten neuen [target_imperator.GetTitle] seit dem Auszug aus Draenor begr��end.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCOE_100;"If we can't return to Draenor... Then we rebuild our civilization in its fullest glory here in this world!" With these words [target_imperator.GetTitledFirstName] started [target_imperator.GetHerHis] speech to the young senate and people of [target_new_goria.GetName].\n\nPurple flags were hung across the capital earlier that day in preparation for this event. Purple and gold banners, lined one after the other, had been placed across the buildings overlooking the central square. The glory of their past radiated from the imperial liverily striking deep pride into the hearts of those assembled. Then the Senate declared the beginning of a new era upon Azeroth, the Gorian Era.\n\nThe masses assembled in the square held their breath. As the Senate finished their decree the people, one by one, started saluting their first new [target_imperator.GetTitle] since the collapse of the previous empire by the orcs.;;�Wenn wir nicht nach Draenor zur�ckkehren k�nnen, werden wir unsere Zivilisation in unserer neuen Welt in vollem Glanz wieder aufbauen.� Mit diesen Worten begann [target_imperator.GetTitledFirstName] [target_imperator.GetHerHis]e Rede vor dem jungen Senat und den Einwohnern von [target_new_goria.GetName].\n\nPurpurne Flaggen wurden fr�her am Tag in der ganzen Hauptstadt zur Vorbereitung dieses einzigartigen Ereignisses gehisst. Der Hauptplatz war von purpurroten und goldenen Bannern umgeben, um den Ruhm des vergangenen und zuk�nftigen Reiches zu demonstrieren.Der Senat erkl�rte den Beginn einer neuen �ra in Azeroth, der gorianischen �ra.\n\nDie Massen versammelten sich auf dem Platz vor dem Senat, lauthals johlend und ihren ersten neuen [target_imperator.GetTitle] seit dem Auszug aus Draenor begr��end.;;;;;;;;;;;x
```

Reply:
reworked, pls review :)